packages/core/src/BackgroundImage/BackgroundImage.stories.args.ts

```diff
@@ -73,4 +73,17 @@ export const argTypes = {
       defaultValue: { summary: undefined },
     },
   },
+
+  backgroundPosition: {
+    name: 'backgroundPosition',
+    type: { name: 'string', required: false },
+    description: 'background position',
+    options: ['top', 'bottom', 'left', 'right', 'center'],
```

Comment:
You should be able to get the array of options as an export from the `BackgroundImage` component file (see my comment there first)